backend/duda/settings/base.py

```diff
@@ -130,6 +130,9 @@ def base_dir_join(*args):
 SENTRY_DSN = config("SENTRY_DSN", default="")
 COMMIT_SHA = config("HEROKU_SLUG_COMMIT", default="")
 
+#Email
+EMAIL = config("EMAIL", default="")
```

Comment:
`FROM_EMAIL` is probably a better variable name here. Remember to update in your .env and Heroku env vars too.